README.md

```diff
@@ -126,15 +126,30 @@ credentials. A subset of the test suite passes without configured AWS
 credentials. To validate your setup, we'll be running one of those tests at the
 end.
 
-1. Create a Python 3.8 virtual environment and activate it:
+1. Activate the `dev` deployment:
+
+   ```
+   cd azul/deployments
+   ln -snf dev .active
+   cd ..
+   ```
+
+2. Load the environment:
+
+   ```
+   source environment
```

Comment:
This runs `scripts/export_environment.py` outside the venv. I think this whole section is an improvement but it's still not 100% right. Maybe `make virtualenv` should NOT depend on `check_env`.